uac-client/src/main/java/com/bazaarvoice/emodb/uac/client/UserAccessControlClient.java

```diff
@@ -30,12 +30,14 @@
 import com.google.common.base.Objects;
 import com.google.common.base.Strings;
 
+import java.io.InputStream;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriBuilder;
 import java.net.URI;
 import java.util.Iterator;
 import java.util.Map;
+import org.glassfish.jersey.client.JerseyClientBuilder;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.glassfish.jersey.client.JerseyClientBuilder'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AUselessImportCheck)